recipes/llamaindex-py-client/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "llamaindex-py-client" %}
+{% set version = "0.1.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/llamaindex_py_client-{{ version }}.tar.gz
+  sha256: 3bd9b435ee0a78171eba412dea5674d813eb5bf36e577d3c7c7e90edc54900d9
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<4.0
+    - poetry-core
+    - pip
+  run:
+    - python >=3.8.0,<4.0.0
+    - pydantic >=1.10
+    - httpx >=0.20.0
+
+test:
+  imports:
+    - llama_index_client
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://llamaindex.ai
+  summary: API definitions for llama-index
+  license: MIT
+  license_file: LICENSE
```

Comment:
Can you add a `dev_url`, please?

Reply:
I don't think there is one :/ it seems to be an autogenerated project that isn't maintained in a GitHub project

Reply:
Meh.